lib/yggdrasil/player/registry.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Yggdrasil.Player.Registry do
+  use GenServer
+  require IEx
+
+  @registry_ets :player_registry_ets
```

Comment:
This probably fine for now, but I notice you are keeping the table id in the state of the process might be worth while to use that instead of the attribute.

I know it's created with a name as well but starting to wonder if this expands to multiple games would be using names?

There could be like a Game ETS that has a name and you can use that to find the table_id for a game (not needed in this process cause it has the id in state, but for external callers).